### PR TITLE
add openwrt init

### PR DESCRIPTION
**Pull request by AndrewMargarit**

### CODE REVIEW

File: examples/openwrt/init.d/socketman.init [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+check_mac()
+{
+        head -1 /etc/mac | grep "-" > /dev/null
+        wc -l "$MACF" | awk '{ print $1 }' | { read wc; test $wc -eq 1; }
+}
+
+start_service()
+{
+        if ! check_mac ; then
+        generate_mac
```
Comment (simonmorley) [inline examples/openwrt/init.d/socketman.init]: Indent please

File: examples/openwrt/init.d/socketman.init [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        procd_open_instance
+        procd_set_param command "$PROG" --config="$CONFIGFILE"
+        procd_set_param respawn 60 5 5
+        procd_set_param stdout 1
+        procd_close_instance
+}
+
+reload_service()
+{
+        echo "Restarting Socketman"
```
Comment (simonmorley) [inline examples/openwrt/init.d/socketman.init]: Capitalize the M

File: examples/openwrt/init.d/socketman.init [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        head -1 /etc/mac | grep "-" > /dev/null
+        wc -l "$MACF" | awk '{ print $1 }' | { read wc; test $wc -eq 1; }
+}
+
+start_service()
+{
+        if ! check_mac ; then
+        generate_mac
+        fi
+        echo "Starting Socketman"
```
Comment (simonmorley) [inline examples/openwrt/init.d/socketman.init]: Capitalize M

File: examples/openwrt/init.d/socketman.init [hunk 4]
```diff
@@ -0,0 +1,46 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2008-2015 OpenWrt.org
+# Copyright (C) 2015 Cucumber WiFi
```
Comment (simonmorley) [inline examples/openwrt/init.d/socketman.init]: Think this is now 17

**Review by simonmorley — CHANGES_REQUESTED** [review]

**Comment by AndrewMargarit** [comment]
I have a newer version.

**Pull request closed without merging**